lib/promoted/ruby/client/options.rb

```diff
@@ -177,8 +177,8 @@ def compact_insertions
           if limit
             insertions_to_compact = insertions_to_compact[0..limit-1]
```

Comment:
i'm thinking i should just remove the limit field so we can use paging.size right away...